config/application-dev.sample.yml

```diff
@@ -40,6 +40,15 @@ spring:
         keycloak:
           truststore:
             certificate: "classpath:[fill-this-value].pem" # certificate file
+  cloud:
+    aws:
+      credentials:
+        access-key: "s3_username"
+        secret-key: "s3_password"
+      s3:
+        region: "dummy"
+        path-style-access-enabled: true
+        endpoint: "http://localhost:9000"
```

Comment:
These have default values in `application.yaml`, should they be put in the dev sample file? The application `s3` entry should be the only block useful by default.

Reply:
I agree ;)